### PR TITLE
chore: add drone build

### PR DESCRIPTION
**Pull request by andrewrynhard**
Signed-off-by: Andrew Rynhard <andrew@andrewrynhard.com>

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  - name: dockersock
+    path: /var/run
+
+steps:
+- name: conform
+  image: autonomy/conform:c539351
+
+- name: build
+  image: autonomy/build-container:latest
+  pull: true
```
Comment (smira) [inline .drone.yml]: I'm not sure, but drone docs have `pull: always`

Comment (andrewrynhard) [reply .drone.yml]: Oh interesting. Fixing.

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+- name: docker
+  image: docker:dind
+  privileged: true
+  volumes:
+  - name: dockersock
+    path: /var/run
+
+steps:
+- name: conform
+  image: autonomy/conform:c539351
```
Comment (bradbeam) [inline .drone.yml:14]: do we want to run a specific tag here or latest / always pull?

**Review by bradbeam — APPROVED** [review]

**Review by smira — APPROVED** [review]

File: .drone.yml (hunk 2 above)
Comment (andrewrynhard) [reply .drone.yml:14]: specific tag for now. at least until we reach another semi-stable version of conform.

**Pull request merged**